### DISCUSSION TITLE
Inventario

### DISCUSSION THREAD

**Discussion by crazyLillo** [tesivo/tesivo_s_VTMB_italian_translation_for_Unofficial_Patch#55] [Generale]
**questa è la modifica di cui parlavo nella discussione "my work"**

![1](https://user-images.githubusercontent.com/96891196/150154005-5ab7ddc0-c8db-44be-b46a-719c1107be6f.jpg)
![2](https://user-images.githubusercontent.com/96891196/150154025-a7b56968-5485-4b92-bd85-c801b3602d8a.jpg)
![3](https://user-images.githubusercontent.com/96891196/150154040-175cff7e-c3ba-4058-aa15-8e3c70615657.jpg)
![4](https://user-images.githubusercontent.com/96891196/150154046-019f45b6-2bd2-44b9-8e76-f2961517cad6.jpg)
![5](https://user-images.githubusercontent.com/96891196/150154056-a659fffc-f0da-44b1-bd79-c7c235e0a4c5.jpg)
![6](https://user-images.githubusercontent.com/96891196/150154070-9991340a-3130-44f1-b92d-06df6513def2.jpg)
![7](https://user-images.githubusercontent.com/96891196/150154082-fc7e34be-6ada-4ffc-9552-c986bdb3701b.jpg)

**Comment by tesivo** [comment]
Ti riferisci alle frasi nei bottoni giusto? Se sì, dovrebbe essere possibile ingrandire le finestre, ma occorre indagare sul file da modificare. Ora non riesco a guardarci, ma appena potrò indagherò.

**Reply by crazyLillo** [reply]
Si, ai bottoni in basso nell' inventario.
In quelle finestrelle entrano max 12 caratteri, io le ho tradotte basandomi sul file originale, cercando di trovare un escamotage traduttorio che non superasse i 12 caratteri.
